modules/debug_cmd/debug_cmd.c

```diff
@@ -84,19 +84,28 @@ static int cmd_ua_debug(struct re_printf *pf, void *unused)
 
 static int cmd_play_file(struct re_printf *pf, void *arg)
 {
+	static struct play *g_play;
 	struct cmd_arg *carg = arg;
 	const char *filename = carg->prm;
-	int err;
-
-	err = re_hprintf(pf, "playing audio file \"%s\" ..\n", filename);
-	if (err)
-		return err;
+	int err = 0;
 
-	err = play_file(NULL, baresip_player(), filename, 0);
-	if (err) {
-		warning("debug_cmd: play_file(%s) failed (%m)\n",
-			filename, err);
-		return err;
+	/* Stop the current tone, if any */
+	g_play = mem_deref(g_play);
+
+	if (str_isset(filename))
```

Comment:
If `filename` will not be set, this block would not change value of `err`. So you are going to return 0. But this means that the function which purpose was to play the file did not perform its action. What about returning some error code?
(It is just an idea)